### ISSUE TITLE
Hardcoded values in setup.py 

### ISSUE THREAD

**Issue by jjungo** [gvalkov/python-evdev#59]
Hello,

In [setup.py](https://github.com/gvalkov/python-evdev/blob/master/setup.py#L69) and [gencodes.py](https://github.com/gvalkov/python-evdev/blob/master/evdev/genecodes.py#L14), locations of `input.h` and `input-event-codes.h` are hardcoded. It seems problematic for cross-compiling.

Example: I use buildroot and try to integrate python-evdev in my local buildroot repo. During the cross-compilation of python-evdev, `gencodes.py` parses theses two headers files in order to generate a `encode.c`. Then the compiler try to compile with the target header files which are located in the buildroot path.

I temporarily changed theses hardocded values to point on my buildroot local repository, but it's ugly and couldn't be automatized.

**Comment by gvalkov** [comment]
Hello @jjungo,

I hope commit ac96a907195245707b671287657a3729d0b9ab13 makes this a bit more straightforward.  Installing with custom header locations should now be possible with:

```
python setup.py build_ext \
  --evdev-headers buildroot/input.h:buildroot/input-event-codes.h \
  --include-dirs  buildroot/ \
  install
```

Kind regards,
Georgi

**Comment by gvalkov** [comment]
This fix has been released as part of python-evdev 0.6.1.
